### ISSUE TITLE
A11y_AzureDataStudioSQL_Project_System Database_ColorContrast: Color contrast ratio of Keyboard focus outline on System Database radio button is less than 3:1

### ISSUE THREAD

**Issue by Yash14j** [microsoft/azuredatastudio#14546]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yWCAG2.1;#A11yTCS;#A11ySev2;#SH_AzureDataStudioSQL_Feb2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG1.4.11;#AINotInScope;#ColorContrast;

### Environment Details:
Application Name: Azure Data Studio
Version: 1.25.3-insider
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Database Project and install extension SQL Database Projects.
2. Tab to SQL Database Project Home Screen with heading "Projects".
3. Tab to connections select your server and connect to it.
4. Few databases will appear under your connection then right click on any database
5. Tab till "Create project from database" and hit enter.
6. New database project will get created under "Projects" tab and tab till "Database references"
7. Right Click and hit enter on "Add Database references".
8. Tab to System Database radio button
9. Start Color Contrast analyser(CCA) tool and analyse color contrast ratio of keyboard focus outline on System Database radio button

### Actual:
Color contrast ratio of Keyboard focus outline on System Database radio button is less than 3:1.

**Same issue is also observed when with Data tier application radio button**

### Expected:
Color contrast ratio of Keyboard focus outline on System Database radio button should be more than or equal to 3:1 

### User Impact:
Low vision users will not be able to see keyboard focus properly if color contrast if low.
